README.md

```diff
@@ -29,7 +29,7 @@ With these deps you can use `ExAws` precisely as you're used to:
 
 ```
 # make a request
-ExAws.S3.list_objects("my-bucket") |> ExAws.request
+ExAws.S3.list_objects("my-bucket") |> ExAws.request(region: "us-west-1")
```

Comment:
Let's have both lines here, I don't want folks to think that the region is required.

Reply:
Good point. Adding the other line back in.